docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -645,7 +645,7 @@
               register: new_reboot
               when: waiting_after_new_reboot.failed
 
-            when: waiting_after_reboot.failed
+            when: waiting_after_reboot.unreachable
```

Comment:
From ansible 2.7 (we are using 2.9), it is possible to use ignore_unreachable,
https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_error_handling.html#ignoring-unreachable-host-errors

In this case, seems that the task status is being set to unreachable instead of failed. 
.unreachable get defined in this scenario and can be used. Below is an e.g.
https://stackoverflow.com/questions/71261197/ansible-change-status-to-failed-when-unreachable